package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "c-slang",
+  "version": "0.0.0",
+  "description": "JavaScript-based implementation of C, written in TypeScript",
+  "main": "index.js",
```

Comment:
If we want to integrate with source academy, it seems like main would be `dist/index` because the app is run from there after building. But I don't really know whether there's a difference.

Reply:
It seems that `main` should be the [primary entry point of the program](https://docs.npmjs.com/cli/v9/configuring-npm/package-json#main). I don't think it's supposed to be a built file. Anyway, I've added an empty `index.js` for now.